WorkflowCombine/Sources/PublisherWorkflow.swift

```diff
@@ -21,17 +21,17 @@ import Foundation
 import Workflow
 
 struct PublisherWorkflow<WorkflowPublisher: Publisher>: Workflow where WorkflowPublisher.Failure == Never {
-    public typealias Output = WorkflowPublisher.Output
-    public typealias State = Void
-    public typealias Rendering = Void
```

Comment:
doing some cleanup while here - `PublisherWorkflow` isn't `public` so these don't need to be either